docs/how-to-guides/index.md

```diff
@@ -14,6 +14,7 @@ tracing/index
 develop-a-flow/index
```

Comment:
refine the order: 

develop-a-flow/index
execute-flow-as-a-function
chat-with-a-flow/index
run-and-evaluate-a-flow/index